webpack.config.js

```diff
@@ -0,0 +1,41 @@
+const path = require('path');
+const BundleTracker = require('webpack-bundle-tracker');
+let LiveReloadPlugin = require('webpack-livereload-plugin');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin')
+
+const mode = process.argv.indexOf("production") !== -1 ? "production" : "development";
+console.log(`Webpack mode: ${mode}`);
+
+module.exports = {
+  entry: './qgisfeedproject/static/js/index',
+  output: {
+    path: path.resolve('./qgisfeedproject/static/bundles'),
+    filename: "bundle.js"
+  },
+  plugins: [
+    new LiveReloadPlugin({appendScriptTag: true}),
```

Comment:
@Xpirix this should only enabled in dev mode

Reply:
@dimasciput Okay, I added the condition